api/query/cache/service/main.go

```diff
@@ -158,12 +159,25 @@ func main() {
 	var netClient = &http.Client{
 		Timeout: time.Second * 5,
 	}
+
+	// Initializes Logger.
+	var gclogClient *gclog.Client
+
+	gclogClient, err = gclog.NewClient(context.Background(), *projectID)
+	if err != nil {
+		logrus.Fatalf("Failed to initiate gclog Client: %v", err)
+	}
+	defer gclogClient.Close()
+
```

Comment:
```suggestion

	// Reuse loggers to prevent leaking goroutines: https://github.com/googleapis/google-cloud-go/issues/720#issuecomment-346199870
```